### ISSUE TITLE
Regressions on ZDEMO_EXCEL42 and ZDEMO_EXCEL_COMMENTS

### ISSUE THREAD

**Issue by sandraros** [abap2xlsx/demos#38]
I just ran `ZDEMO_EXCEL_CHECKER` and I see two regressions:
- [`ZDEMO_EXCEL42`](https://github.com/abap2xlsx/demos/blob/main/src/zdemo_excel42.prog.abap)
  ![image](https://github.com/abap2xlsx/demos/assets/34005250/00594293-39a1-414e-80e2-7c6865df7e62)
  Clicking on `theme1.xml` opens Visual Code but no difference is shown. In fact, there are lots of trailing spaces after some of the XML elements of the old version, which are ignored by the checker, and no trailing spaces in the new version. I feel it's due to https://github.com/abap2xlsx/abap2xlsx/pull/1046.
- [`ZDEMO_EXCEL_COMMENTS`](https://github.com/abap2xlsx/demos/blob/main/src/zdemo_excel_comments.prog.abap)
  ![image](https://github.com/abap2xlsx/demos/assets/34005250/73ac20b4-9889-406b-9f82-3e3e65c9bca5)
  I guess it was due to #34.

Probably, regenerating the reference Excel files of these 2 programs in the Web repository will be fine, via `ZDEMO_EXCEL_CHECKER`:
![image](https://github.com/abap2xlsx/demos/assets/34005250/b3611ec1-e1e1-4e4f-9573-e6064bcccd51)

**Comment by AndreaBorgia-Abo** [comment]
Checking the file properties of theme1.xml (for Excel42) under AppData\Local\SAP\SAP GUI\tmp [before](https://github.com/AndreaBorgia-Abo/abap2xlsx/releases/tag/demos_38_excel42-pre) and [after](https://github.com/AndreaBorgia-Abo/abap2xlsx/releases/tag/demos_38_excel42-post) I see they're both 7905 bytes long but I suppose this is expected after the comparison cleanup. Indeed it is only during items compare that the untouched files are present.

For the Comments demo, discounting the theme1.xml difference above, the change ([before](https://github.com/AndreaBorgia-Abo/abap2xlsxdemos/releases/tag/demos_38_comments-pre) / [after](https://github.com/AndreaBorgia-Abo/abap2xlsxdemos/releases/tag/demos_38_comments-post)) is indeed responsible for the differences shown.

TLDR ok to update the reference files!

**Comment by AndreaBorgia-Abo** [comment]
fixed by #40
